x-pack/plugins/translations/translations/fr-FR.json

```diff
@@ -10391,7 +10391,6 @@
     "xpack.alerting.breadcrumbs.createMaintenanceWindowsLinkText": "Créer",
     "xpack.alerting.breadcrumbs.editMaintenanceWindowsLinkText": "Modifier",
     "xpack.alerting.breadcrumbs.maintenanceWindowsLinkText": "Fenêtres de maintenance",
-    "xpack.alerting.breadcrumbs.stackManagementLinkText": "Gestion de la Suite",
```

Comment:
the translations can be restored now that the key is being used again, right?

Reply:
yes, i ll add them back